### PR TITLE
Update UserGuide with Deletet and Addt

### PR DESCRIPTION
**Pull request by choonyongchan**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -261,6 +262,50 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+4. The `parse` function of FindCommandParse will parse the input and instantiate a new `Predicate` object for each
+   argument present. At the end, all `Predicate` objects are combined using the `MultiplePredicate` object.
+
+5. The command communicates with the `Model` to filter out the found person from the existing AddressBook.
+
+6. The result of the command execution is encapsulated as a `CommandResult` object which is returned back from `Logic`.
+
+The following sequence diagram shows how the FindCommand function works:
+
+![UpdatedAddCommandSeqDiagram](images/FindCommandDiagram.png)
```
Comment (bklimey) [inline docs/DeveloperGuide.md:291]: File not commited in this PR

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -261,6 +262,50 @@ The following activity diagram summarizes what happens when a user executes a ne
...
+
+The following activity diagram summarizes what happens when a user executes a new command:
+
+![UpdatedAddCommand](images/UpdatedFindCommand.png)#
```
Comment (bklimey) [inline docs/DeveloperGuide.md:295]: File not commited in this PR

**Review by bklimey — COMMENTED** [review]
Files not found in PR. And minor nit, please seperate DG update in another PR next time as the title of this PR is just UG update. Thanks.

**Review by hsiaotingluv — APPROVED** [review]
Other than BK's feedback, Good to go :))

**Pull request merged**